packages/ui/src/server/controllers/availability.ts

```diff
@@ -53,7 +53,13 @@ function getAvailability(req: Request, clusters: Record<string, {id: string}>, o
 }
 
 export async function getClustersAvailability(req: Request, res: Response) {
-    const odinPath = getApp().config?.odinBaseUrl;
+    const {ytAuthCluster: cluster} = req.params;
+    const odinPath = _.get(
```

Comment:
As I understand `odinPath` should be calculated for each cluster in `getAvailability`, shouldn't it?